### ISSUE TITLE
About more training detail

### ISSUE THREAD

**Issue by bedman367** [R00Kie-Liu/TA2N#2]
I' very interested at your training detail, and in the paper it says"For more details about training strategies (e.g. optimizer, learning rate), please refer to supplementary material."  
But I can't find the related supplementary material, could you provide more details?
Thanks !

**Comment by R00Kie-Liu** [comment]
Thanks for your attention! The supplementary material is attached to our Arxiv version.

I'll release the training & evaluation part of TA2N this month as soon as possible (too busy recently ><
